src/Repository/Doctrine/MessageRepository.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace Adshares\AdsOperator\Repository\Doctrine;
+
+use Adshares\AdsOperator\Document\Message;
+use Adshares\AdsOperator\Repository\MessageRepositoryInterface;
+
+class MessageRepository extends BaseRepository implements MessageRepositoryInterface
+{
+    /**
+     * @return array
+     */
+    public function availableSortingFields(): array
+    {
+        return [
+            'id',
+            'blockId',
+        ];
+    }
+
+    /**
+     * @param string $messageId
+     * @return Message|null
+     * @throws \Doctrine\ODM\MongoDB\LockException
+     * @throws \Doctrine\ODM\MongoDB\Mapping\MappingException
+     */
+    public function getMessage(string $messageId):? Message
+    {
+        /** @var Message $message */
+        $message = $this->find($messageId);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Immediately return this expression instead of assigning it to the temporary variable "$message". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=php%3AS1488)